Tests/SentryProfilerTests/SentryThreadMetadataCacheTests.mm

```diff
@@ -47,8 +47,6 @@ - (void)testRetrievesThreadMetadata
         SENTRY_PROF_LOG_ERROR_RETURN(pthread_setschedparam(thread, policy, &param));
     }
 
-    std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
`m`: One CI job failed https://github.com/getsentry/sentry-cocoa/actions/runs/5483653476/jobs/9990243216?pr=3134 after removing the sleeps. Maybe we still need them.

Reply:
yeah looks like it! just much shorter than 1 second